server/routes/validators/reviewPlan/reviewNoteValidator.ts

```diff
@@ -1,13 +1,13 @@
 import type { ReviewNoteForm } from 'reviewPlanForms'
 import formatErrors from '../../errorFormatter'
-import textValueExceedsLength from '../../../validators/textValueValidator'
+import { isEmpty, textValueExceedsLength } from '../../../validators/textValueValidator'
 
 const MAX_REVIEW_NOTE_LENGTH = 512
 
 export default function validateReviewNote(reviewNoteForm: ReviewNoteForm): Array<Record<string, string>> {
   const errors: Array<Record<string, string>> = []
 
-  if (!reviewNoteForm.notes) {
+  if (isEmpty(reviewNoteForm.notes)) {
```

Comment:
Call the new validator function (and we can do the same thing for other validators like goal title, step title etc; but I've chosen not to do those in this PR)